blocks/contact-form/contact-form.js

```diff
@@ -20,30 +25,59 @@ export default function decorate(block) {
       div.classList.add('contact-form__field');
     }
 
+    // Check for field name and add a fitting input "type" attribute and assign variables
+    let firstName;
+    let lastName;
+    let email;
+    let birthDate;
+    let termsOfService;
```

Comment:
we talked about how not maintainable this solution is, because it's too explicitly tied to the current requirements.
you don't have to think about all possible future expansions of the component, but you should think about a few things:

1. make sure the supported field types (text, email, checkbox and date) can be used to add additional fields
1. eventually (not at this stage, but when it's all rounded up and you're just polishing things), make sure that the supported field types, if they have extra validation or range options (like with date), can also be added/defined somehow